src/pages/workspaces/workspace/analysis/ContextBar.js

```diff
@@ -201,11 +196,11 @@ export const ContextBar = ({
           tooltipSide: 'left',
           disabled: !isTerminalEnabled,
           href: terminalLaunchLink,
-          onClick: () => {
-            Ajax().Metrics.captureEvent(Events.analysisLaunch,
+          onClick: async () => {
+            await Ajax().Metrics.captureEvent(Events.analysisLaunch,
               { origin: 'contextBar', source: 'terminal', application: 'terminal', workspaceName, namespace })
-            if (window.location.hash === terminalLaunchLink && currentRuntime?.status === 'Stopped') {
-              startCurrentRuntime()
+            if (currentRuntime?.status === 'Stopped') {
+              Ajax().Runtimes.runtimeWrapper(currentRuntime).start()
```

Comment:
You need to add some error handling around this Ajax call (note that `captureEvent` is wrapped in `withErrorIgnoring` which is why it didn't require error handling).